ddp-workspace/projects/ddp-sdk/src/lib/directives/sticky-scroll.directive.ts

```diff
@@ -0,0 +1,145 @@
+import { AfterViewInit, OnDestroy, Directive, ElementRef, HostListener, Renderer2 } from '@angular/core';
+import { WindowRef } from '../services/windowRef';
+
+// todo: consider replace with position fixed with sticky once we stop supporting IE11 (check if it simplifies the logic)
```

Comment:
I guess we should not take care about IE11,
but it is worth double-checking.

Reply:
@MocanaAtBroad thoughs? I saw ie11 is oficially deprecated in angular but they are going to support it until November 2022. https://angular.io/guide/deprecations#internet-explorer-11

Reply:
We don't care about IE at all. And particularly regarding Prism!

Reply:
I tried to apply position sticky but it turned out it doesn't work together with overflow: auto (even with overflow-x: auto), so unfortunately we can't use position sticky in this directive. I removed the todo comment